geomesa-gt/geomesa-gt-partitioning/src/test/scala/org/locationtech/geomesa/gt/partition/postgis/PartitionedPostgisDataStoreTest.scala

```diff
@@ -109,14 +109,37 @@ class PartitionedPostgisDataStoreTest extends Specification with BeforeAfterAll
   }
 
   "PartitionedPostgisDataStore" should {
+    "not work" in {
+      val ds = DataStoreFinder.getDataStore(params.asJava)
+      ds must not(beNull)
+
+      try {
+        ds must beAnInstanceOf[JDBCDataStore]
+
+        // This sft name exceeds 31 characters, so it should fail
+        val sft = SimpleFeatureTypes.renameSft(this.sft, "abcdefghijklmnopqrstuvwxyzabcde_____")
+        ds.getTypeNames.toSeq must not(contain(sft.getTypeName))
+        ds.createSchema(sft) must throwAn[java.io.IOException].like {
+          case e => e.getMessage mustEqual "Error occurred creating table"
+        }
+      } catch {
+        case NonFatal(e) => logger.error("", e); ko
```

Comment:
```suggestion
```
can probably delete this and just let the error bubble up - alternatively there is a `fail(e)` method in specs that will pass in the exception.